venv_update.py

```diff
@@ -307,6 +307,10 @@ def adjust_options(options, args):
     argv[:] = ('virtualenv',) + args
     info(colorize(argv))
     raise_on_failure(virtualenv.main)
+    # There might not be a venv_path if doing something like "venv= --version"
+    # and not actually asking virtualenv to make a venv.
+    if return_values.venv_path is not None:
+        run(('rm', '-rf', join(return_values.venv_path, 'local')))
```

Comment:
There's a test that does this:

```
ckuehl@xenialdev1-uswest1cdevc:~/proj/venv-update$ venv-update venv= --version
> virtualenv --version
15.1.0
```

I guess we want to preserve that functionality? /shrug